elixir/apps/web/lib/web/live/policies/new.ex

```diff
@@ -134,6 +134,13 @@ defmodule Web.Policies.New do
       |> put_default_params(socket)
       |> map_condition_params(empty_values: :drop)
 
+    params =
```

Comment:
Do you remember what was the error? That `conditions` can not be empty?

Reply:
Yes, the IP allowlist fails validation.